See: https://gitlab.gnome.org/GNOME/gnome-builder/-/issues/2160

From 7aaaecefc2ea8a37eaeae8b4d726d119d4eb8fa3 Mon Sep 17 00:00:00 2001
From: Christian Hergert <chergert@redhat.com>
Date: Mon, 15 Jan 2024 11:54:38 -0800
Subject: [PATCH] libide/tweaks: remove autoptr cleanup

This is not needed now with recent GTK changes merged this morning.
---
 src/libide/tweaks/ide-tweaks-window.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/libide/tweaks/ide-tweaks-window.c b/src/libide/tweaks/ide-tweaks-window.c
index f2553a396e..dcb55bb351 100644
--- a/src/libide/tweaks/ide-tweaks-window.c
+++ b/src/libide/tweaks/ide-tweaks-window.c
@@ -32,8 +32,6 @@
 #include "ide-tweaks-panel-list-private.h"
 #include "ide-tweaks-window.h"
 
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (GtkStackPage, g_object_unref)
-
 struct _IdeTweaksWindow
 {
   AdwWindow               parent_instance;
-- 
GitLab

